tests/conftest.py

```diff
@@ -118,32 +116,32 @@ def _app_factory(config=None):
                     hold=remote_hold_handler,
                     renew=remote_renew_handler,
                 )
-            ),
-            test_invalid=dict(
-                login_handler=remote_login_failed_handler,
-            ),
+            )
         )
     )
-    Babel(app)
-    InvenioDB(app)
-    InvenioAccess(app)
-    InvenioAccounts(app)
+    Babel(app_)
```

Comment:
I do not understand why the app name has been changed.

Reply:
oops, it's a mistake